lambda.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can we still use run.py and import it here? lambda.py will be just an entrypoint.